marimbabot_planning/src/move_to_pose.cpp

```diff
@@ -24,15 +24,16 @@ moveit::planning_interface::MoveGroupInterface::Plan concatinated_plan(std::vect
     assert(plans.size() > 0);
 
     moveit::planning_interface::MoveGroupInterface::Plan plan{plans[0]};
-    ros::Duration time_from_start{0};
+    ros::Duration time_from_start{1};
```

Comment:
Why did you do this? @berkgungor semantically zero is correct. Also, what is `time_added`?